lib/hyrax/transactions/steps/validate.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+require 'dry/monads'
+
+module Hyrax
+  module Transactions
+    module Steps
+      ##
+      # Saves a given work, returning a Result (Success|Failure)
```

Comment:
The documentation does not match the implementation. It appears that we are validating the change_set, and not saving the given work.

Reply:
oops! good catch.